kubernetes/kube-lego/base/configmap.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
 metadata:
   name: kube-lego
```

Comment:
Is this now going to be in the `kube-system` namespace?

Reply:
Negative, but the `kube-lego` namespace declaration moved to the prod overlay